dbt/include/trino/macros/materializations/table.sql

```diff
@@ -6,8 +6,8 @@
       {%- set on_table_exists = 'rename' -%}
   {% endif %}
   {%- set identifier = model['alias'] -%}
-  {%- set tmp_identifier = model['name'] + '__dbt_tmp' -%}
-  {%- set backup_identifier = model['name'] + '__dbt_backup' -%}
+  {%- set tmp_identifier = identifier ~ '__dbt_tmp' -%}
```

Comment:
Can we create the target relation first and leverage dbt's implementation instead of fiddling with alias ourselves?

Reply:
refactored